### PR TITLE
367 Use IA config for identification, fixed metadata error too

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- IA config data used for Identification rather than hardcoding
- metadata validation code duplication fixed
- Tests updated to use valid algorithm for zebra
- Fixed bug in identification jobs handling

---

### CODE REVIEW

**Comment by drewblount** [comment]
LGTM

File: tests/tasks/app/test_job_control.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def test_sighting_identification_jobs(
         id_configs = [
             {
                 'algorithms': [
-                    'Pie',
+                    'hotspotter_nosv',
                 ],
                 'matchingSetDataOwners': 'mine',
             }
```
Comment (colinwkingen) [inline tests/tasks/app/test_job_control.py:145]: Reminds me to make a ticket to grab an ID config for a given ia_class.

**Review by colinwkingen — APPROVED** [review]
LGTM.

**Pull request merged**